### ISSUE TITLE
Eigen minimum version

### ISSUE THREAD

**Issue by jmirabel** [stack-of-tasks/pinocchio#694]
Currently, the minimum version is 3.0.4 and it is not correct. I know that 3.2.4 does not work and 3.2.92 does. Is it fine to update the minimum version to 3.2.92 (ubuntu 16.04) ?

**Comment by nim65s** [comment]
I think Pinocchio is still working on 14.04 with eigen 3.2.0

**Comment by jmirabel** [comment]
@ksyy had this error with 3.2.4:
```
/usr/localDev/HPP/install/include/pinocchio/multibody/joint/joint-revolute.hpp:157:63:
error: could not convert ‘Eigen::DenseBase<Derived>::Zero(Eigen::DenseBase<Derived>::Index)
  [with Derived = Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::Matrix<double, 3, 1> >;
           Eigen::DenseBase<Derived>::ConstantReturnType = Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::Matrix<double, 3, 1> > >;
            typename Eigen::internal::traits<T>::Scalar = double;
            Eigen::DenseBase<Derived>::Index = long int]()’
from ‘const ConstantReturnType
  {aka const Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::Matrix<double, 3, 1> > >}’
to ‘pinocchio::TransformRevoluteTpl<double, 0, 0>::LinearType
  {aka Eigen::CwiseNullaryOp<Eigen::internal::scalar_constant_op<double>, Eigen::Matrix<double, 3, 1> >}’
     LinearType translation() const { return LinearType::Zero(3); };
```

**Comment by nim65s** [comment]
https://travis-ci.org/stack-of-tasks/pinocchio/builds/496407364 : job .1 is trusty, and job .2 is xenial,

**Comment by jmirabel** [comment]
Then, likely, there is one function which is not unit-tested.

**Comment by jcarpent** [comment]
The minimal version is 3.0.5.

**Comment by jmirabel** [comment]
@jcarpent Then, there is a bug at `multibody/joint/joint-revolute.hpp:157`

**Comment by jmirabel** [comment]
It can be repeated by compiling hpp-pinocchio devel branch, with branch topic/fcl-release of Pinocchio and Eigen 3.2.4.

**Comment by jcarpent** [comment]
The main question is in the end why @ksyy is working with 3.2.4? Because not LTS is furnished with such version.

**Comment by nim65s** [comment]
This is the version of robotpkg-eigen3, so I suppose he got it from there, but I have no idea why this one was installed on his desktop.

**Comment by jmirabel** [comment]
It was unintended. It was a forgotten installed version which is now deleted. I opened the issue for reporting. It is not blocking us.

**Comment by jcarpent** [comment]
Solved by #695.
